app/controllers/themes_controller.rb

```diff
@@ -21,6 +21,18 @@ def create
     end
   end
 
+  def edit
```

Comment:
1. edit just need to pass the theme to view 2.missing the update step. that is where you need to add the update